e2e/cart/userCanBuyProducts.spec.ts

```diff
@@ -0,0 +1,34 @@
+import {expect, test} from '@playwright/test';
+import {loginTestHelper, productNames, productNamesAfterRemoveIndex1} from '../../helpers/common';
+import {ProductsPage} from '../../pages/products/ProductsPage';
+import {ProductPage} from '../../pages/products/ProductPage';
+import {CartPage} from '../../pages/cart/CartPage';
+import {PaymentDataPage} from '../../pages/cart/PaymentDataPage';
+import {PaymentInformationPage} from '../../pages/cart/PaymentInformationPage';
+
+test.describe('User can add items in cart, delete items and buy products', async () => {
+  test('User can buy products', async ({page}) => {
+    await loginTestHelper(page, 'standard_user');
+
+    const productsPage = new ProductsPage(page);
+    const cartPage = new CartPage(page);
+    for (let productName of productNames) {
+      await productsPage.addToProduct(productName);
+    }
+    await productsPage.goToCart();
+    await cartPage.checkYourCartTextIsVisible();
+    await expect(await cartPage.getProductTitle(), 'Product title is not correct').toEqual(productNames);
+    await cartPage.removeProduct('Sauce Labs Bike Light');
+    await expect(await cartPage.getProductTitle(), 'Product title is not correct').toEqual(productNamesAfterRemoveIndex1);
```

Comment:
why is there a comparison of an array of 5 elements, where, in theory, there should be left 2 elements